src/main/java/godaddy/registry/jtoolkit2/se/fee/DomainRestoreFeeResponseExtension.java

```diff
@@ -51,13 +51,14 @@ public void fromXML(XMLDocument xmlDoc) throws XPathExpressionException {
             for (int feeNodeIndex = 0; feeNodeIndex < feeNodes.getLength(); feeNodeIndex++) {
                 Node feeNode = feeNodes.item(feeNodeIndex);
                 String feeNodeValue = feeNode.getTextContent();
-                String feetype = feeNode.getAttributes().getNamedItem("description").getTextContent();
-
-                if (feetype.equalsIgnoreCase("Restore Fee")) {
-                    restoreFee = new BigDecimal(feeNodeValue);
-                }
-                if (feetype.equalsIgnoreCase("Renewal Fee")) {
-                    renewFee = new BigDecimal(feeNodeValue);
+                if (feeNode.getAttributes().getNamedItem("description") != null) {
```

Comment:
Same comment as the other place - could you please extract a var for `feeNode.getAttributes().getNamedItem("description")`, and use it in both line 54 and 55?

Reply:
Done